src/app/content/App/Notice/NoNotice/NoNoticeScreen.tsx

```diff
@@ -50,19 +51,25 @@ export default () => (
       <NoNotice />
     </Image>
     <Title>
-      Pas encore de post sur <br /> cette page web
+      <Trans i18nKey={'contributions.disclaimer_no_post'} />
     </Title>
     <Container>
       <Content>
         <AddNoticeButton />
-        <Paragraph>une info, un conseil</Paragraph>
+        <Paragraph>
+          <Trans i18nKey={'contributions.button_subtext'} />
+        </Paragraph>
       </Content>
-      <Separator>ou</Separator>
+      <Separator>
+        <Trans i18nKey={'noun.or'} />
+      </Separator>
       <Content>
         <BackgroundButton as={ReactRouterDomLink} to="/question">
-          Demander
+          <Trans i18nKey={'action.ask'} />
```

Comment:
Prefer `t` syntax for simple content.
> https://react.i18next.com/getting-started#simple-content

Reply:
No problem.
But in this case it will be necessary to import the two syntax. Trans : for line 54, because there is HTML in the sentence and t for what is left, okay?

Reply:
I suppose it's okay.